functions.php

```diff
@@ -13,3 +13,54 @@ function design_system_public_enqueue_global_styles() {
 }
 
 add_action( 'enqueue_block_assets', 'design_system_public_enqueue_global_styles' );
+
+
+/**
+ * Registers block pattern categories for the Design System theme.
+ * This function checks if the register_block_pattern_category_type function exists,
+ * then registers a custom block pattern category type. It also defines an array of
+ * block pattern categories and applies a filter to allow other plugins or themes to
+ * modify the block pattern categories. Finally, it registers each block pattern category.
+ *
+ * @since 0.8.0
+ */
+function design_system_register_block_patterns() {
+
+	if ( function_exists( 'register_block_pattern_category_type' ) ) {
+		register_block_pattern_category_type(
+			'bc-gov-logo-light',
+			array(
+				'label' => __( 'Design System Theme', 'bc-gov-logo-light' ),
```

Comment:
I don't think these are the intended arguments for `__()`: https://developer.wordpress.org/reference/functions/__/.